js/tab.js

```diff
@@ -46,7 +46,7 @@ define([
   };
   
   //hopefully this never screws up unaugmented Ace sessions.
-  Tab.prototype = EditSession.prototype;
+  Object.assign(Tab.prototype, EditSession.prototype);
```

Comment:
Had to update this, otherwise `EditSession.prototype.$startWorker.call(this)` just pointed at the function we just overrode it with.